lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -676,6 +677,62 @@ DWARFASTParserClang::ParseTypeModifier(const SymbolContext &sc,
   case DW_TAG_volatile_type:
     encoding_data_type = Type::eEncodingIsVolatileUID;
     break;
+  case DW_TAG_LLVM_ptrauth_type: {
+    DWARFDIE ptr_die = die.GetReferencedDIE(DW_AT_type);
```

Comment:
Would be nice to put all of this in a helper function, in order to not bloat the size of `ParseTypeModifier`. But feel free to ignore.

Reply:
Done, thanks, see eecbb370d3d0257a5a4ffb644448219d41aa5426